extensions/ql-vscode/src/view/common/SuggestBox/__tests__/SuggestBox.test.tsx

```diff
@@ -200,6 +200,9 @@ describe("SuggestBox", () => {
     });
 
     await userEvent.click(screen.getByRole("combobox"));
+
+    expect(screen.getAllByRole("option")).not.toHaveLength(0);
+
     await userEvent.keyboard("{Enter}");
 
     expect(screen.queryByRole("option")).not.toBeInTheDocument();
```

Comment:
Do we need to `waitFor` this one too?

Reply:
It doesn't seem like it. I haven't seen this fail for Enter, only for Tab.